### ISSUE TITLE
Errors in Jest do not line up with source

### ISSUE THREAD

**Issue by turt2live** [vector-im/riot-web#11524]
This actually fails on line 157 of the compiled source, which is indeed the test it named. The code snippet it provides is for a completely different test.

```
Summary of all failing tests
FAIL spec/unit/crypto.spec.js (12.523s)
  ● Crypto › Session management › restarts wedged Olm sessions

    Timeout - Async callback was not invoked within the 5000ms timeout specified by jest.setTimeout.

      155 |                         });
      156 |                     });
    > 157 |                 };
          |         ^
      158 |
      159 |                 const keyshareEventForEvent = async (event, index) => {
      160 |                     const eventContent = event.getWireContent();

      at Spec (node_modules/jest-jasmine2/build/jasmine/Spec.js:85:20)
      at Suite.<anonymous> (spec/unit/crypto.spec.js:157:9)
      at Suite.<anonymous> (spec/unit/crypto.spec.js:72:5)
      at Object.<anonymous> (spec/unit/crypto.spec.js:57:1)

```

**Comment by t3chguy** [comment]
I believe this is fixed in newer Jest versions, we are blocked until your Typescript/babel changes land before we can switch to it

**Comment by turt2live** [comment]
That's mildly annoying because it only adds to how much I hate the JS ecosystem.

**Comment by turt2live** [comment]
Closing in favour of https://github.com/vector-im/riot-web/issues/8880
